tests/Behat/Context/FixtureContext.php

```diff
@@ -74,4 +74,43 @@ public function contentBlocksAreNotInLineEditable()
 
         $this->getMainContext()->visit('/?flush');
     }
+
+    /**
+     * @Given the page template includes elemental blocks
+     */
+    public function pageTemplateIncludesElementalBlocks()
+    {
+        $template = <<<'TEMPL'
+<h1>$Title</h1>
+<div class="content">
+    $Content
+    $ElementalArea
+</div>
+TEMPL;
+
+        $templFile = 'Layout/Page.ss';
+        $templPath = $this->getDestinationTemplatePath($templFile, true);
+        file_put_contents($templPath, $template);
```

Comment:
Where will this file go?  Into the current projects template path?  That doesn't seem right if that's the case.

Reply:
It gets added to `app/templates/Layout/Page.ss`
I'm open to other suggestions - I don't want to override the simple theme template 'cause then I have to restore it correctly afterwards - I guess I could create a temporary theme instead... but I don't see what the difference is at the end of the day.

Reply:
Are there any other behat tests that add template files where we could copy their approach?

Reply:
Not that I could find, that's why I need https://github.com/silverstripe/silverstripe-behat-extension/pull/214 as well.